### PR TITLE
Ado limited pat

### PR DESCRIPTION
**Pull request by Fleid**
Context : https://www.notion.so/dbtlabs/ADO-Limited-PAT-05747193879f472797d5010d7b6f0c0a

https://github.com/dbt-labs/dbt-cloud/commit/187f76f3daed6a8aca3838aba78039d2c359593a

### CODE REVIEW

**Review by sidpatel13 — APPROVED** [review]
LGTM!

File: website/docs/docs/cloud/git/setup-azure.md [hunk 1]
```diff
@@ -374,7 +374,10 @@ A dbt Cloud account admin with access to the service user's Azure DevOps account
...
 
-The Azure DevOps Administrator can limit the creation of full-scoped PATs by enabling a policy that restricts users to a custom-defined set of scopes. By default, this policy is set to **off**, but enabling it will cause project setup to fail with an error. After disabling this policy and successfully setting up your project, if you wish to use finer-scoped permissions, some features such as webhooks for CI may be lost, so we recommend the service user has full-scoped PATs.  To exclude the dbt Cloud service user from the global PAT policy, add them to the allow list as part of your security policy.
+These tokens are limited to the following [scopes](https://learn.microsoft.com/en-us/azure/devops/integrate/get-started/authentication/oauth?view=azure-devops):
+- `vso.code_full` : Grants full access to source code, metadata about commits, changesets, branches, and other version control artifacts. Also grants the ability to create and manage code repositories, create and manage pull requests and code reviews, and to receive notifications about version control events via service hooks. Also includes limited support for Client OM APIs.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/git/setup-azure.md]:
```suggestion
- `vso.code_full`: Grants full access to source code and version control metadata (commits, branches, and so on). Also grants the ability to create and manage code repositories, create and manage pull requests and code reviews, and receive notifications about version control events with service hooks. Also includes limited support for Client OM APIs.
```

File: website/docs/docs/cloud/git/setup-azure.md [hunk 2]
```diff
@@ -374,7 +374,10 @@ A dbt Cloud account admin with access to the service user's Azure DevOps account
...
+- `vso.project` : Grants the ability to read projects and teams.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/git/setup-azure.md]:
```suggestion
- `vso.project`: Grants the ability to read projects and teams.
```

File: website/docs/docs/cloud/git/setup-azure.md [hunk 3]
```diff
@@ -374,7 +374,10 @@ A dbt Cloud account admin with access to the service user's Azure DevOps account
...
+These tokens are limited to the following [scopes](https://learn.microsoft.com/en-us/azure/devops/integrate/get-started/authentication/oauth?view=azure-devops):
+- `vso.code_full` : Grants full access to source code, metadata about commits, changesets, branches, and other version control artifacts. Also grants the ability to create and manage code repositories, create and manage pull requests and code reviews, and to receive notifications about version control events via service hooks. Also includes limited support for Client OM APIs.
+- `vso.project`: Grants the ability to read projects and teams.
+- `vso.build_execute` : Grants the ability to access build artifacts, including build results, definitions, and requests, and the ability to queue a build, update build properties, and the ability to receive notifications about build events via service hooks.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/git/setup-azure.md]:
```suggestion
- `vso.build_execute`: Grants the ability to access build artifacts, including build results, definitions, and requests, and the ability to queue a build, update build properties, and the ability to receive notifications about build events with service hooks.
```

File: website/docs/docs/cloud/git/setup-azure.md [hunk 4]
```diff
@@ -374,7 +374,10 @@ A dbt Cloud account admin with access to the service user's Azure DevOps account
 Once connected, dbt Cloud displays the email address of the service user so you know which user's permissions are enabling headless actions in deployment environments. To change which account is connected, disconnect the profile in dbt Cloud, sign into the alternative Azure DevOps service account, and re-link the account in dbt Cloud.
 
 :::info Personal Access Tokens (PATs)
-dbt Cloud generates temporary access tokens called Full-scoped PATs for service users to access APIs related to their dbt Cloud project. These tokens are only valid for a short period of 5 minutes and become invalid after they are used to make an API call.
+dbt Cloud leverages the service user to generate temporary access tokens called [PATs](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?toc=%2Fazure%2Fdevops%2Fmarketplace-extensibility%2Ftoc.json&view=azure-devops&tabs=Windows). These tokens are limited in scope, and only valid for a short period of 5 minutes and become invalid after they are used to make an API call.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/git/setup-azure.md]:
```suggestion
dbt Cloud leverages the service user to generate temporary access tokens called [PATs](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?toc=%2Fazure%2Fdevops%2Fmarketplace-extensibility%2Ftoc.json&view=azure-devops&tabs=Windows). 

These tokens are limited in scope, are only valid for 5 minutes, and become invalid after a single API call.
```

**Review by mirnawong1 — APPROVED** [review]
some small tweaks and thank you for opening this up @Fleid ! happy to merge this for you, please let me know if you have strong feelings about the tweak and we can make further changes!

**Pull request merged**